passes/cmds/scatter.cc

```diff
@@ -46,22 +46,19 @@ struct ScatterPass : public Pass {
 		CellTypes ct(design);
 		extra_args(args, 1, design);
 
-		for (auto &mod_it : design->modules_)
+		for (auto module : design->selected_modules())
 		{
-			if (!design->selected(mod_it.second))
-				continue;
-
-			for (auto &c : mod_it.second->cells_)
-			for (auto &p : c.second->connections_)
+			for (auto cell : module->cells())
+			for (auto &p : cell->connections_)
```

Comment:
This could also be `connections()`

Reply:
Indeed, although I'm not sure b6eaff69dc28fa2dffd4b0572ca73b0ef823e143 counts as much of a cleanup :-\.